src/main/java/io/cryostat/jfr/datasource/server/Datasource.java

```diff
@@ -80,11 +80,22 @@ void root(RoutingContext context) {
         response.end();
     }
 
-    @Route(path = "/search", methods = HttpMethod.GET)
+    @Route(path = "/search", methods = HttpMethod.POST)
     void search(RoutingContext context) {
         HttpServerResponse response = context.response();
-        setHeaders(response, "application/json", "GET");
-        response.end(recordingService.search());
+        setHeaders(response, "application/json", "POST");
```

Comment:
Rather than setting headers manually/explicitly here, is it possible/does it make sense to use:

1. [Quarkus CORS filter](https://quarkus.io/guides/http-reference#cors-filter)
2. `Route(produces = { ReactiveRoutes.APPLICATION_JSON })`

Reply:
Oh right! I will look into this and update to use Quarkus magic :D

Reply:
> [Quarkus CORS filter](https://quarkus.io/guides/http-reference#cors-filter)

Just quick notice that grafana and datasource is deployed in the same pod. Do we need to set up CORS? All queries in grafana dashboard front-end sends requests to grafana backend first anyways.

Reply:
![Screenshot from 2022-09-29 17-20-34](https://user-images.githubusercontent.com/68053619/193144195-2bc7d10e-8225-4a6b-949a-c585d757c2d1.png)

Reply:
Some settings from Grafana:

https://github.com/tthvo/cryostat-grafana-dashboard/blob/30ddbdbc180766a2582d6875480cfa8f7debc468/datasource.yaml#L9

Looks like we specify `proxy` for server request instead of browser so we can drop `CORS`?

Reply:
Makes sense to me.

Reply:
Fixed that :D